insights/parsers/etcd_conf.py

```diff
@@ -0,0 +1,89 @@
+"""
+EtcdConf - file ``/etc/etcd/etcd.conf``
+=======================================
+
+The EtcdConf class parses the file ``/etc/etcd/etcd.conf``. The
+etcd.conf is in the standard 'ini' format and is read by the base
+parser class `IniConfigFile`.
+
+Typical contents of the file look like::
+
+[member]
```

Comment:
The following error is reported when building the doc:
~~~
/tmp/workspace/insights-core_PR-2111-AFDYMPSFBPHBDBTWN2K6WM4EF4DKBXKS43MIAXBH4GSSNAOZ5U5A/insights/parsers/etcd_conf.py:docstring of insights.parsers.etcd_conf:11:Inconsistent 
~~~
It can be fixed by adding one intent tab (4 spaces) to the following typical contents,

Reply:
Thanks @xiangce. Fixed the indentation :)